tests/context-menu.spec.ts

```diff
@@ -14,6 +14,8 @@ async function moveMouseAwayFromTable(page: Page) {
 }
 
 test.beforeEach(async ({ page }) => {
+  await page.goto('');
+  await expect(page.locator('.loading-spinner')).toHaveCount(0);
```

Comment:
Since we're doing the exact same thing in a dozen spots now, we should put this in a utility method we can call. In utils.ts, something like:
```
async function gotoPage(url: string, options?: ...): Promise<void> {
  await page.goto(url, options);
  await expect(...)
}
```